chapters/ch43.md

```diff
@@ -1,45 +1,45 @@
 > # 43. Artificial data synthesis
 
---> _replace THIS LINE by your translation for the above line_
+# 43. Tổng hợp dữ liệu nhân tạo
 
 
 > Your speech system needs more data that sounds as if it were taken from within a car. Rather than collecting a lot of data while driving around, there might be an easier way to get this data: By artificially synthesizing it.
 
---> _replace THIS LINE by your translation for the above line_
+Hệ thống nhận diện giọng nói của bạn cần dữ liệu những âm thanh được lấy từ trong xe hơi. Thay vì thu thập nhiều dữ liệu trong khi lái xe xung quanh, có thể có một cách làm dễ dàng hơn để có được dữ liệu này: Bằng cách tổng hợp một cách nhân tạo.
```

Comment:
> Hệ thống ~~nhận diện~~ giọng nói của bạn cần _thêm_ dữ liệu ~~những âm thanh~~ _nghe giống như được lấy từ trong xe hơi. Thay vì thu thập nhiều dữ liệu trong khi lái xe ~~xung quanh~~, ~~có thể~~ có một cách ~~làm~~ dễ dàng hơn để ~~có~~ _lấy_ được dữ liệu này: Bằng cách tổng hợp một cách nhân tạo.

Câu đầu có thể gọn lại thành "Hệ thống giọng nói của bạn cần thêm dữ liệu nghe giống như được lấy từ trong xe hơi". Từ "nhận diện" có vẻ thừa ở đây vì hệ thống đó có thể không phải để nhận *diện*. Cụm "that sounds as if" có nghĩa thô là "mà chúng nghe như thể là", từ "sounds" ở đây là "nghe như" (động từ) chứ không phải là "những âm thanh" (danh từ).

Bỏ từ "xung quanh" trong câu thứ hai. Từ "around" chỉ chung chung là lái "qua lại".